toolchest/gen_gitreview.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import os
+
+
+def generate_gitreview(path, host, host_port, project,
```

Comment:
I agree that it should be in the previous order, I think I just changed it because I matched the order of the arguments with the order of the parts of the file written (so it was really just moving the project parameter after the host)